src/tss2-tcti/tcti-tbs.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-2 */
+/*
+ * Copyright (c) 2018 Intel Corporation
+ * All rights reserved.
+ */
+#ifndef TCTI_TBS_H
+#define TCTI_TBS_H
+
+#include "tcti-common.h"
+
+#define TBS_RESULT_MAX_BUFFER_SIZE 8192
```

Comment:
The result will never be bigger then 4K. The other 4K will never be used.

Reply:
This should be fixed now, I have changed TBS_RESULT_MAX_BUFFER_SIZE to 4096.